test-lib.sh

```diff
@@ -539,7 +577,7 @@ EOF
         echo "CMD /usr/libexec/s2i/run" >>"$df_name"
     fi
     # Run the build and tag the result
-    docker build -f "$df_name" --no-cache=true -t "$dst_image" .
+    docker build $(ct_mount_ca_file) -f "$df_name" --no-cache=true -t "$dst_image" .
```

Comment:
Please do not make the volume argument specific for this use case. Let's future proof and parse `$s2i-args` for any volume definitions and pass them to `docker build` here.